### PR TITLE
Update to english version

### PR DESCRIPTION
**Pull request by neihousaigaai**
Due to the issue #106

### CODE REVIEW

File: json/role_info.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "name_vi": "Thần tình yêu",
     "description": {
       "vi": "Đầu mỗi ván chơi, Cupid sẽ được gọi dậy và chọn ra hai người yêu nhau. Cặp đó sẽ chết nếu 1 trong 2 bị chết. Nếu hai người thuộc hai phe khác nhau (Sói vs Dân) thì họ thành phe thứ 3 với nhiệm vụ là hai người cuối cùng sống sót.",
-      "en": ""
+      "en": "Ships 2 players."
```
Comment (nitro2) [inline json/role_info.json:96]:
```suggestion
      "en": "On the first day, Cupid can choose 2 players and make them to be a couple. If one of them is dead, the other will be also dead. If the couple belongs to different parties (Werewolf and Villager), then they become the third party. Their mission is to be the last stand players."
```

File: json/role_info.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
     "name_vi": "Phù thủy",
     "description": {
       "vi": "Vào ban đêm, Phù thủy có 1 bình phép có thể phục sinh một người chơi đã chết, đôi khi cũng có thể nguyền rủa một người còn sống. Mỗi chức năng này chỉ được sử dụng một lần duy nhất trong game.",
-      "en": ""
+      "en": "Can reborn a dead player, and also can curse an alive player."
```
Comment (nitro2) [inline json/role_info.json:78]:
```suggestion
      "en": "Can reborn a dead player, and also can curse an alive player once per game."
```

File: json/role_info.json [hunk 3]
```diff
@@ -66,7 +66,7 @@
     "name_vi": "Cáo",
     "description": {
       "vi": "Thuộc phe thứ ba, cáo sẽ chiến thắng nếu tất cả sói bị giết. Sẽ chết nếu bị Tiên tri soi trúng. Tuy nhiên, Cáo không chết nếu được Bảo vệ trong đêm bị soi.",
-      "en": ""
+      "en": "A fox."
```
Comment (nitro2) [inline json/role_info.json:69]:
```suggestion
      "en": "A fox belongs to third party. Fox wins if all werewolves are killed. Fox is dead if being revealed by Seer. Fox will not be dead if being protected by Guard."
```

File: json/role_info.json [hunk 4]
```diff
@@ -57,7 +57,7 @@
     "name_vi": "Kẻ phản bội",
     "description": {
       "vi": "Là người nhưng bị mua chuộc theo phe sói. Kẻ phản bội biết ai là sói nhưng phe sói không biết ai là Kẻ phản bội.",
-      "en": ""
+      "en": "A human but betrays Villagers."
```
Comment (nitro2) [inline json/role_info.json:60]:
```suggestion
      "en": "Betrayer. A human but betrays Villagers. Betrayer knows werewolf players."
```

File: json/role_info.json [hunk 5]
```diff
@@ -21,16 +21,16 @@
...
     "daytime_command": "vote",
     "nighttime_commands": ["kill"]
   },
   "Guard": {
     "name_vi": "Bảo vệ",
     "description": {
-      "vi": "Bảo vệ được chọn 1 người khác nhau mỗi đêm trừ bản thân và người được chọn sẽ bất tử đêm đó.",
-      "en": ""
+      "vi": "Bảo vệ được chọn 1 người khác nhau mỗi đêm và người được chọn sẽ bất tử đêm đó.",
+      "en": "Guards a player."
```
Comment (nitro2) [inline json/role_info.json:33]:
```suggestion
      "en": "Guards a player: Protect the player from being killed."
```

**Comment by nitro2** [comment]
Does this finish? Don't forget to remove DRAFT in title for final version.

**Comment by neihousaigaai** [comment]
> Does this finish?

The game status embed template hasn't completed yet.

**Comment by nitro2** [comment]
Look like this change corrupt `test.py`. I'm fixing it.

**Pull request merged**